tests/Unit/Utilities/Test_Tuple.cpp

```diff
@@ -21,28 +21,55 @@ struct tuple_fold_plus {
   }
```

Comment:
Can you please separate out formatting this file from the changes you're making? This is rather difficult to parse